spaces/tokens.json

```
@@ -152,5 +152,6 @@
   "0xeaccb6E0f24d66cF4Aa6cBDa33971b9231d332a1": "polyient-games-governance",
   "0xaea8e1b6cb5c05d1dac618551c76bcd578ea3524": "sogur",
   "0x683EcFe5B148c8840998c1B4884F60a8eAD432D1": "d5",
-  "0xa150db9b1fa65b44799d4dd949d922c0a33ee606": "digital-reserve-currency"
+  "0xa150db9b1fa65b44799d4dd949d922c0a33ee606": "digital-reserve-currency",
+  "0xfe9a29ab92522d14fc65880d817214261d8479ae": "snow"
```

please do not edit this file.
```suggestion
  "0xa150db9b1fa65b44799d4dd949d922c0a33ee606": "digital-reserve-currency"
```